cmake/projects/Boost/hunter.cmake

```diff
@@ -21,6 +21,18 @@ hunter_add_version(
     PACKAGE_NAME
     Boost
     VERSION
+    "1.59"
+    URL
+    "http://sourceforge.net/projects/boost/files/boost/1.59.0/boost_1_59_0.tar.gz/download"
```

Comment:
You can use these releases:
* https://github.com/hunter-packages/boost/releases/tag/v1.60.0
* https://github.com/hunter-packages/boost/releases/tag/v1.59.0